.gitignore

```diff
@@ -3,3 +3,4 @@
 bin
 build*
 *.*.h
+.vscode
```

Comment:
This should go in your local `.git/info/exclude` (or an excludesFile like `~/.gitignore`)